### DISCUSSION TITLE
Will deleting a user affect previously submitted code?

### DISCUSSION THREAD

**Discussion by roy-ltp** [community/community#113394] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Will deleting a user affect previously submitted code?

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @roy-ltp! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

With that being said, deleting your personal account removes all repositories, forks of private repositories, wikis, issues, pull requests, and pages owned by your account. Issues and pull requests you've created and comments you've made in repositories owned by other users will not be deleted. Your resources and comments will become associated with the [ghost user](https://github.com/ghost). You can learn more here: [Deleting your personal account](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-personal-account/deleting-your-personal-account).

I've gone ahead and moved this to the correct category for you. Good luck!
